primitive/coq_float64.c

```diff
@@ -72,3 +74,13 @@ DECLARE_FUNOP(next_down, next_down(x))
 value coq_is_double(value x) {
   return Val_long(Is_double(x));
 }
+
+#define uint63_of_value(val) ((uint64_t)(val) >> 1)
+
+double coq_uint63_to_float(value x) {
+  return (double) uint63_of_value(x);
+}
+
+value coq_uint63_to_float_byte(value x) {
+  return caml_copy_double(coq_uint63_to_float(x));
+}
```

Comment:
I really don't know why I wrote this like that but the easiest solution is to remove that C code and use the OCaml primitive `float_of_int` in `uint63_63.ml`, that is replace there the following lines
```OCaml
external to_float : int -> (float [@unboxed])
  = "coq_uint63_to_float_byte" "coq_uint63_to_float"
[@@noalloc]
```
by
```OCaml
let to_float = float_of_int
```
Sorry about that